.github/dependabot.yml

```diff
@@ -4,7 +4,14 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    versioning-strategy: "increase-if-necessary"
   - package-ecosystem: "npm"
     directory: "/"
     schedule:
       interval: "daily"
+    allow:
+      - dependency-name: "@stoplight/elements-web-components"
```

Comment:
I had a deja vu with this file, but in fact I already reviewed that part elsewhere :wink:

Reply:
I messed up and based this off another PR branch not master. My bad :\